docs/blog/lion-without-polyfills.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Lion without poylfills
+published: true
+description: Lion has been a long user of the scoped registry - always requiring a polyfill - but no more. Load the polyfill only if you need it.
+date: 2022-04-04
+tags: [javascript, polyfills]
+cover_image: /blog/images/introducing-lions-website-cover-image.jpg
+---
+
+From the start we have been focused on making it very explicit which sub component are used within a given custom element.
+This is commonly revered to as [component composition](https://lit.dev/docs/composition/component-composition/).
+On top of that we scoped these [sub components](https://open-wc.org/docs/development/scoped-elements/) to the [current shadow root](https://github.com/WICG/webcomponents/blob/gh-pages/proposals/Scoped-Custom-Element-Registries.md).
+
+This means that you can use a lion component like `lion-listbox` and never have to worry about if the internally used components clash with others you are already using.
+
+## How does it work?
+
+1. Within lion class files we only import other class files
+2. We define them as `scopedElements` and let the mixin handle the rest
+
+To clarify within lion class files we never import files that run `customElement.define`
+
+```js
+import { LitElement, html } from 'lit';
+import { ScopedElementsMixin } from '@open-wc/scoped-elements';
+import { LionOptions } from './LionOptions.js';
+
+export class LionListbox extends ScopedElementsMixin(LitElement) {
+  static scopedElements = {
+    'lion-options': LionOptions,
+  };
+
+  render() {
+    return html`
+      <div>
+        <!-- example code - not the real implementation -->
+        <lion-option></lion-option>
+      </div>
+    `;
+  }
+}
+```
+
+## What is wrong with it?
```

Comment:
The title seems to suggest smth is wrong with the above implementation, whereas it was the prev version of ScopedElementsMixin was wrong.

Maybe make it "## Known challenges in previous releases"